db/db.go

```diff
@@ -3,11 +3,14 @@ package db
 import (
 	"context"
 	"database/sql"
+	"errors"
 	"io/ioutil"
 	"path"
 	"runtime"
 )
 
+var ErrDealNotFound = errors.New("deal not found")
```

Comment:
If this error is being used for storage and funds as well as deals, let's call it ErrNotFound